code_sharing/client/test/widget_test.dart

```diff
@@ -0,0 +1 @@
+
```

Comment:
Not even a smoke test to confirm it compiles? Sad face

Reply:
Oh right, that's fair!